webapp/src/filework/mod.rs

```diff
@@ -0,0 +1,35 @@
+#![allow(unused)]
+use std::fs::{File, remove_file};
+use std::io::{Write, Seek, SeekFrom};
+use std::env;
+use std::path::{PathBuf, Path};
+
+pub fn write_source_to_file(source_code: &str, lang: &str) -> PathBuf
+{
+    // If error ever happens on work with temp files it's not on the user, 
+    // so he should get "internal server error" here
+    let mut input_file_name = env::var("COMPILATION_TEMP_DIR")
+        .expect("COMPILATION_TEMP_DIR doesn't exist");
+    input_file_name.push_str("/");
+    input_file_name.push_str(
+        &chrono::Utc::now()
+        .format("compilation_input-%Y-%m-%d-%H-%M-%S-%f").to_string()
```

Comment:
Is it possible to reuse `generate_filename` somehow?

Reply:
> Is it possible to reuse `generate_filename` somehow?

I'm sorry, could you explain the word "reuse"? Did you mean using this function twice in one lib and another lib in the same time?